packages/@stylexjs/babel-plugin/src/shared/utils/__tests__/generate-css-rule-test.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow strict
+ */
+
+import { generateCSSRule } from '../generate-css-rule';
+
+describe('generateCSSRule', () => {
```

Comment:
Is there a reason we need these extra tests instead of relying on the existing public API tests? The vast majority of them snapshot generated CSS metadata already